src/workerd/api/util.h

```diff
@@ -58,6 +58,7 @@ kj::Maybe<kj::ArrayPtr<const char>> readContentTypeParameter(kj::StringPtr conte
 // Assumptions:
 //   - `contentType` has a single Content-Type parameter, e.g. "boundary" or "charset".
```

Comment:
This is no longer true, right?

Reply:
My main issue is the use of the word "single".

Reply:
Improved comment.